src/Tmds.Ssh/SshClientSettings.Defaults.cs

```diff
@@ -63,6 +65,40 @@ private static IReadOnlyList<Credential> CreateDefaultCredentials()
         return credentials.AsReadOnly();
     }
 
+    private static List<Name> CreatePreferredEncryptionAlgorithms()
+    {
+        // Prefer AesGcm over ChaCha20Poly when the platform has AES instructions.
+        bool addAesGcm = AesGcm.IsSupported;
+        bool hasAesInstructions =  System.Runtime.Intrinsics.X86.Aes.X64.IsSupported ||
```

Comment:
Shouldn't we be checking based on the process architecture and not all of them?

Reply:
I think we can just check all of them. When we're checking for a different architecture, the `IsSupported` will return `false`.